Modules/Microsoft365DSC/DSCResources/MSFT_AADVerifiedIdAuthority/settings.json

```diff
@@ -0,0 +1,29 @@
+{
+    "resourceName": "AADVerifiedIdAuthority",
+    "description": "This resource configures an Azure AD Verified Identity Authority.",
+    "permissions":    {
+    "graph":  {
+                  "delegated":  {
+                                    "read":  [
+                                                 {
+                                                     "name":  "VerifiableCredential.Authority.ReadWrite"
+                                                 }
+                                             ],
+                                    "update":  [
+
+                                               ]
```

Comment:
Update permissions are required, even if they are the same as the ones for Read.

Reply:
The permission I added wasn't related to msgraph so I have removed it from settings.json and added it readme.md for now